react/src/lib/EventOverlay/index.js

```diff
@@ -607,10 +607,10 @@ class EventOverlay extends React.Component {
           ) {
             targetNode.style.bottom = `${documentScrollTop + windowBottom - documentBottom}px`;
           }
-          if (elementDims.right > documentRight || this.elementWidth > documentRight) {
+          if (elementDims.right >= documentRight || this.elementWidth > documentRight) {
```

Comment:
By this point in the code the overlay box has already been placed. If it is right at the edge of the page it will have the same right value as documentRight hence the change to >=